SU2_CFD/include/solvers/CTurbSolver.hpp

```diff
@@ -79,6 +84,28 @@ class CTurbSolver : public CSolver {
    */
   inline CVariable* GetBaseClassPointerToNodes() final { return nodes; }
 
+private:
+
+  /*!
+   * \brief Compute the viscous flux for the turbulent equation at a particular edge.
+   * \param[in] iEdge - Edge for which we want to compute the flux
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] numerics - Description of the numerical method.
+   * \param[in] config - Definition of the particular problem.
+   */
+  void Viscous_Residual(unsigned long iEdge,
+                        CGeometry *geometry,
+                        CSolver **solver_container,
+                        CNumerics *numerics,
+                        CConfig *config);
```

Comment:
I fused the viscous and convective residual loops of the turbulent solvers in the same way I did for CEuler and CNSSolver. This makes the reducer strategy more cost-effective as we do not need to clear the Jacobian matrix and rhs vector during preprocessing (we overwrite instead).